src/js/components/TaskTable.js

```diff
@@ -30,8 +30,8 @@ class TaskTable extends React.Component {
     return task.resources[prop];
   }
 
-  getStateValue(task, prop) {
-    return TaskStates[task[prop]].displayName;
+  getStatusValue(task) {
+    return TaskStates[task['state']].displayName;
```

Comment:
or we could even `getStatusValue({state}) { return TaskStates[state].displayName; }` if we wanted to be fancy